### DISCUSSION TITLE
Cannot use Sandboxie because of the reminder

### DISCUSSION THREAD

**Discussion by Ralf-AM** [sandboxie-plus/Sandboxie#2892] [General]
I had installed Sandboxie-Plus about half year ago. If i now start sandboxie it always shows me a supporting reminder that i cannot get rid off because the "continue" button is always grrayed out (i only can click on "input certificate" and "close) [sorry if it is not the correct translation of the buttons because i installed the german version]
I also fully uninstalled sandboxie-plus, removed all registry items with "sandboxie" in it and re-installed the version 1.9.3 (x64). But nothing helps and i again get the reminder popup with "You have installed Sandboxie-Plus more than 132 days ago"

![Screenshot 1  2023-05-12_002732](https://github.com/sandboxie-plus/Sandboxie/assets/1440458/c6873c9c-f0f1-4a5b-bfb0-05675d6b26aa)

Can anyone please tell me how to get sandboxie-plus working again - i thought its free for personal use so why can't i click on "continue" to close the reminder popup and continue using sandboxie?

**Comment by isaak654** [comment]
It appears that you previously selected the commercial option in the setup wizard dialog.

It can be verified before the expiration date from the Sandbox menu -> Maintenance -> Setup Wizard:

<img alt="Setup Wizard" src="https://github.com/sandboxie-plus/Sandboxie/assets/12372772/2451963b-8b51-4eaa-b52c-ec238c790b6b" height="250px">

In my case, I cannot switch directly from the personal use option to the commercial use option (unless I request a new certificate).

**Reply by bastik-1001** [reply]
In the selected language that is: Sandbox menu -> Wartung -> Einrichtungsassitent

**Comment by DavidXanatos** [comment]
@Ralf-AM the question is why have you selected business use when you are using it personally?
Anyhow, to rectify this you need to enter a personal use certificate, this will switch it back to personal use.
If you absolutely don't want to get one you can email me to get a 3 day personal test certificate, its only good for 3 days but should be enough to switch it to personal use.
